src/platform/StdSchedulerPoll.cpp

```diff
@@ -143,13 +143,12 @@ bool StdScheduler::DoScheduleProcs(int iTimeout)
 		bool any_executed = false;
 		auto tNow = C4TimeMilliseconds::Now();
 		// Which process?
-		for (size_t i = 0; i < procs.size(); i++)
+		for (auto proc : procs)
```

Comment:
I observed the similarity. Please provide your thoughts on [my comment](https://github.com/openclonk/openclonk/pull/41#discussion_r113731696).